### DISCUSSION TITLE
[4.x]: Use native field group for default title field

### DISCUSSION THREAD

**Discussion by JeanLucEsser** [craftcms/cms#10898] [Ideas] [answered]
I never use the default title field for entries. When designing the layout of a section, I choose my own field for the title so I can for example set a character count on it or more. But my reason for it is not the point.

I get why we are offered a default title. We are not all advanced users and for 90% of the people using Craft it's great.

But for the rest of us, I've always been annoyed by the fact that un selecting title on the layout designer page didn't remove (visually that is) the title from the layout, but only marked it as hidden. I know, kind of a silly OCD. But still.

That is until Craft 4. I had the pleasure, and I think it's new to Craft 4, to see a default "Native" field group for an Alt field in the assets designer. Treating default titles the same way as native fields would be great. It's a great place to put them when we don't want them. Plus it would allow you to have more "defaults" fields for beginners should you want to go that way.

Best,
Adrian Monk ;)

**Comment by brandonkelly** [comment, marked as answer]
“Title” fields are implemented as Native fields already. We just don’t have a good way of removing them from the field layout if the entry type states that a custom title format should be used instead. So we show that it will be “hidden” instead. Agree that the UI could be improved though.

**Reply by JeanLucEsser** [reply]
That’s what I’m saying. 

You do have a way now in Craft 4. Look at how you treated the Alt field in the assets layout designer. You created a default  « Native fields » fields group. I don’t remember it being there in Craft 3 but maybe I never noticed it. Anyway that looks like the right way to do it. 

<img width="282" alt="image" src="https://user-images.githubusercontent.com/14908712/162900485-87280675-0f43-49cd-ac41-82f2ca732507.png">

**Reply by brandonkelly** [reply]
Alternative Text just isn’t configured as “mandatory” within the field layout. We have always had the option to do that with Title as well, however we don’t have a way, currently, of showing/hiding the Title Format setting depending on whether the field is included in the field layout.
